### DISCUSSION TITLE
Layout suggestion

### DISCUSSION THREAD

**Discussion by WreckageFather** [URUWorks/TeroSubtitler#306] [Ideas]
This is currently my biggest issue trying to go from SW to Tero: unless I'm missing a setting, there is currently no way to make the entry list resemble that of Subtitle Workshop's, i.e. short, compact, one-line-tall entries. As it is now, in Vertical layout entries are so tall that no more than 4 can fit on my screen at a time, so a legacy style list would be very welcome.

**Comment by chenlung** [comment]
Could you supply an image of Subtitle Workshop and how it currently looks in Tero Subtitler?

**Reply by WreckageFather** [reply]
Sure thing!
![SubtitleWorkshop_Q1VCiizpwa](https://github.com/URUWorks/TeroSubtitler/assets/166918365/4cfe6cf5-f6f2-41d8-bdce-68e7a5a37d10)
![tero_iCL4SORByY](https://github.com/URUWorks/TeroSubtitler/assets/166918365/f14c838c-4ff2-406d-a50f-d20c9a46c191)

**Comment by Mantrhax** [comment]
SW looks a lot better tham tero

**Reply by chenlung** [reply]
Hi, @Mantrhax.

I'm sure the UX and UI could be improved (I'm hoping professionals can step forward with opinions and even pull requests), but could you be more specific?
